### PR TITLE
Add error message when toggleBlocks() fails

### PR DESCRIPTION
**Pull request by cpirich**
- Added an error message dialog to handle the error that occurs when you try to "Show Blocks" after creating code in the ace editor that can't be parsed as JavaScript
- Tweaked the StubDialog class that handles dialogs in the console output when the dashboard is not hosting our apps

### CODE REVIEW

File: apps/src/feedback.js [hunk 1]
```diff
@@ -748,6 +748,35 @@ exports.showGeneratedCode = function(Dialog) {
   dialog.show();
 };
 
+exports.showToggleBlocksError = function(Dialog) {
```
Comment (davidsbailey) [inline apps/src/feedback.js]: Function params should be lower/camel case, maybe studioDialog to avoid name collision?

**Comment by davidsbailey** [comment]
LGTM, you may choose to ignore the style nit since you're following the style in that file. Would be awesome to show a dialog when a JS error is encountered while running the program as well, rather than showing "success / retry / share" dialog.

File: apps/src/feedback.js (hunk 1 above)
Comment (cpirich) [inline apps/src/feedback.js]: yeah, I followed the convention, which we may want to clean up.

**Comment by cpirich** [comment]
I added some better error handling a few weeks back where it will actually select the code that fails during execution and it outputs info to the console output window. Good feedback on improving the dialog as well.

**Pull request merged**